### ISSUE TITLE
files missing from "make install"

### ISSUE THREAD

**Issue by umlaeute** [iainctduncan/scheme-for-pd#4]
`make install` installs `scm4pd.pd_linux` and the `README.md`.

However, a couple of files are not installed, including:
- `patchers/s4pd-help.pd` (nitpitck: in Pd world the term "patcher" is not very common (only among those that have converted from Max; the main help-patch could probably just go into the root directory anyhow)
- `scm/*.scm` (i'm not sure if all the files in here ought to be included or not. but at least the helpfiles depends on `s4pd.scm`, so that should probably be installed)

~~~Makefile
datafiles += \
  patchers/s4pd-help.pd \
  $(wildcard scm/*.scm) \
  $(empty)
~~~

**Comment by iainctduncan** [comment]
thanks. Can you clarify,  I add that snippet to the Makefile?
I will remove the patchers dir and just put help at top level.

**Comment by umlaeute** [comment]
yes.

Fixed with d4596b9a3f2ee654975f5dca44de5ddc81dfb8da
